### ISSUE TITLE
BUG Problem with m3u on the android version of the emulator.

### ISSUE THREAD

**Issue by FrensisMcBlob** [Rakashazi/emu-ex-plus-alpha#1605]
### Describe the bug

Does not open CHD images via m3u.

### Steps to reproduce

Just run any image through m3u.

### Expected behavior

Should have started the game.

### Add screenshots

![Screenshot_20240324-172538](https://github.com/Rakashazi/emu-ex-plus-alpha/assets/87436296/a2358c0b-aeae-46d0-933e-c9da3cef758b)

**Comment by Rakashazi** [comment]
What version of Android are you using?

**Comment by FrensisMcBlob** [comment]
Android 9 Pie

ср, 3 апр. 2024 г. в 11:52, Robert Broglia ***@***.***>:

**Comment by TheBoxGame** [comment]
@Rakashazi fix mapper mmc3 in version 1.5.68 of nes.emu the rom from the 45 MMC3 mapper worked normally and after updating the mmc3 mapper the rom does not work as it should https://github.com/Rakashazi/emu-ex-plus-alpha/issues/1610

**Comment by FrensisMcBlob** [comment]
Waiting for a fix?

ср, 3 апр. 2024 г. в 12:44, TheBoxGamePL ***@***.***>:

**Comment by TheBoxGame** [comment]
@FrensisMcBlob yes

**Comment by TheBoxGame** [comment]
@FrensisMcBlob it is already fixed

**Comment by TheBoxGame** [comment]
I'll tell you what I mean about the problem, see since version 1.5.68 the rom with the 45 MMC3 mapper worked normally and now it doesn't work even when I run the game
![319078649-85142dc0-4ee5-419c-82c2-6b6b9a0c74fc](https://github.com/Rakashazi/emu-ex-plus-alpha/assets/109180532/5f34b9aa-359b-4f0f-b431-4619dda06bad)
![319078661-d0fdad81-92cf-4fe4-9201-4f45c4be8156](https://github.com/Rakashazi/emu-ex-plus-alpha/assets/109180532/dddd4683-c965-4a13-8f08-9182c6fa085c)
![319078746-75a9d487-743e-4275-9701-33df73ee1187](https://github.com/Rakashazi/emu-ex-plus-alpha/assets/109180532/800aa951-8f9d-477b-a80c-c42d7d1521b0)

**Comment by TheBoxGame** [comment]
https://github.com/Rakashazi/emu-ex-plus-alpha/assets/109180532/ed034709-8c82-4f5f-a7d2-07a5fd702ec6

**Comment by FrensisMcBlob** [comment]
If everything is already fixed, when is the update
on the marketplace?

чт, 4 апр. 2024 г. в 11:09, TheBoxGamePL ***@***.***>:

**Comment by Rakashazi** [comment]
@FrensisMcBlob I think I found the m3u issue, please test again in the current nightly when it builds.

**Comment by FrensisMcBlob** [comment]
Yes, the nightly version now runs m3u, but hangs and crashes after a couple
minutes.

ср, 10 апр. 2024 г. в 15:05, Robert Broglia ***@***.***>:

**Comment by FrensisMcBlob** [comment]
Version 1.5.80 now runs the game with m3u normally. BUT, there is no choice of disk loading, i.e. it starts, but only the first disk, the second or third disk cannot be selected.
